cert_issuer/certificate_handlers.py

```diff
@@ -121,7 +121,7 @@ def get_certificate_generator(self):
 
     def finish_batch(self, tx_id, chain):
         proof_generator = self.merkle_tree.get_proof_generator(tx_id, chain)
-        for metadata in self.certificates_to_issue.items():
+        for _, metadata in self.certificates_to_issue.items():
```

Comment:
It would be nice to add a test for this :). I notice if I undo this line in your PR the test-suite still passes..